src/pydocstyle/checker.py

```diff
@@ -680,6 +681,10 @@ def check(filenames, select=None, ignore=None, ignore_decorators=None):
         try:
             with tokenize_open(filename) as file:
                 source = file.read()
+                try:
+                    source = source.decode('utf-8')
+                except:
```

Comment:
I think this except should be more narrowly scoped. There are a surprising number of exceptions that can happen here and they depend on platform, how the python interpreter is compiled, etc.